### PR TITLE
[db_bench] Enable write rate limit for updaterandom benchmark

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Summary:
We have FLAGS_benchmark_write_rate_limit to limit write rate in
db_bench, but it was not in use for updaterandom benchmark.

Test Plan:
Run the benchmark with FLAGS_benchmark_write_rate_limit on and check the
printed write rate.

### CODE REVIEW

File: tools/db_bench_tool.cc [hunk 1]
```diff
@@ -489,7 +489,8 @@ static bool ValidateCacheNumshardbits(const char* flagname, int32_t value) {
   return true;
 }
 
-DEFINE_bool(verify_checksum, true, "Verify checksum for every block read"
+DEFINE_bool(verify_checksum, true,
+            "Verify checksum for every block read"
```
Comment (yiwu-arbug) [inline tools/db_bench_tool.cc]: side effect of clang-format.

File: tools/db_bench_tool.cc [hunk 2]
```diff
@@ -4592,6 +4593,12 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         abort();
       }
 
+      if (thread->shared->write_rate_limiter) {
+        thread->shared->write_rate_limiter->Request(
+            key.size() + value_size_, Env::IO_HIGH, nullptr /*stats*/,
+            RateLimiter::OpType::kWrite);
+      }
```
Comment (sagar0) [inline tools/db_bench_tool.cc]: Looking at other uses of write_rate_limiter in this file, it seems like more needs to be done. 
For example, if you look at `TimeSeriesWrite`, it seems to be doing: 
```
    if (FLAGS_benchmark_write_rate_limit > 0) {
      write_rate_limiter.reset(
          NewGenericRateLimiter(FLAGS_benchmark_write_rate_limit));
    }
```
Don't you need to do something like this?

Comment (sagar0) [inline tools/db_bench_tool.cc]: May be I am wrong, but don't you need to check `FLAGS_benchmark_write_rate_limit` also and handle it, and not just if `thread->shared->write_rate_limiter` is not null?

Comment (sagar0) [inline tools/db_bench_tool.cc]: nah, forget it. I'm wrong. Digging further I realized that all the things I mentioned above are already handled in `RunBenchmark` function.

Comment (yiwu-arbug) [inline tools/db_bench_tool.cc]: Yeah, maybe TimeSeriesWrite is using it in a wrong way..

**Pull request closed without merging**